.github/workflows/backend-cd.yml

```
@@ -113,6 +113,7 @@ jobs:
           docker run --name momo-api
           --rm -d -p 80:8080
           --volume=$HOME/security:/momo/security:ro
+          --volume=$HOME/logs:/momo/logs
```

쓰기 권한임을 명시해야할 것 같아요!
```suggestion
          --volume=$HOME/logs:/momo/logs:rw
```

---

`$HOME` 은 기본 환경변수로, `~/` 경로를 나타냅니다.
현재 배포된 서버 기준으로 절대경로는 `/home/ubuntu` 경로가 되겠네요.

설정해 주신대로라면 호스트 머신의 `/home/ubuntu/logs` 경로가, 컨테이너 내의 `/momo/logs` 경로로 마운트됩니다.